golden_data.Rmd

```diff
@@ -1,56 +1,90 @@
+### UniProt
 
-![](images/amoA_AOA_download.png){width=100%}
+The UniProt database architecture is built upon standard biological attributes, in contrast to the *de novo* clustering and classification scheme of EggNOG. Protein sequences are therefore searchable using multiple keys covering a range of taxonomic and functional specificities. The most specific is the unique identifier of a single protein sequence (e.g. Q04507 is an AmoA sequence in Nitrosomonas europaea). More broad searchable attributes include protein names, gene names, organism, [Enzyme Commission (EC)](https://en.wikipedia.org/wiki/Enzyme_Commission_number) numbers, and identifiers of other databases such as PFam and TIGRFAM.
 
-Deselect "Aligned" then click the "Download" button.
+Here, we will use the gene names AmoA and PmoA to find reference sequences in UniProt. Begin by navigating to [https://www.uniprot.org/](https://www.uniprot.org/) and typing "Particulate methane monooxygenase" (including the double quotation marks) into the search bar at the top of the page.
 
-Repeat with the above steps with the other genes "amoA_AOB" and "PmoA".
+#### Refining the search
 
-### EggNOG
+12,231 sequences were returned by the initial query. In the following figure, we can see many entries were unrelated to pmoA (e.g. ftsH, ffsA, carB).
 
-Go to the EggNOG website at http://eggnog5.embl.de/#/app/home and search for either "ENOG5028JPK" or "arCOG08676" in the search bar at the top of the page. These EggNOG ortholog identifiers correspond to PmoA/AmoA from Bacteria and AmoA from Archaea, respectively. In the image below, we've searched for "ENOG5028JPK".
+![](images/UniProt_pmo.png){width=100%}
+To remove these off-target search results, we will need to expand our search terms and for this we will turn to the "Advanced" drop-down menu on the right side of the search bar. Add "pmoa" for the *Gene name [GN]* and remove all sequences with unresolved lineages by specifying "uncultured bacterium" for *Taxonomy [OC]* with the *NOT* logical operator. The latter filter was applied since these poorly annotated sequences are typically derived from short environmental samples and are not taxonomically informative. The advanced search fields should look identical to the figure below.
 
-![](images/EggNOG_xmoa_1.png){width=100%}
+![](images/pmoA_advanced_search.png){width=100%}
 
-Click "Download" at the bottom-left of the panel and select "All _XX_ sequences (FASTA)", where _XX_ is the number of sequences belonging to the orthologous group (OG). This will open a new window with all sequences from that OG in FASTA format. Download this file to a file on your computer following the format &lt;OG name&gt;_EggNOGv5.faa where &lt;OG name&gt; is the EggNOG ortholog identifier.
+This more specific search should yield 3882 entries with the vast majority being predicted pmoA sequences.
```

Comment:
maybe fudging this number is more sustainable (ex. ~3k). The number is now 3885.